apps/yapms/src/lib/utils/applyPanZoom.ts

```diff
@@ -18,17 +18,34 @@ function lockMap(lock: boolean) {
 	}
 }
 
-function applyPanZoom(svg: SVGElement) {
+function applyPanZoom(svg: SVGSVGElement) {
 	if (panZoomSettings !== undefined) {
 		panZoomSettings.panzoom.dispose();
 	}
+	const bb = svg.getBBox();
+	svg.setAttribute('viewBox', '0 0 ' + (bb.x + bb.width + bb.x) + ' ' + (bb.y + bb.height + bb.y));
+	let handler: NodeJS.Timeout | undefined;
+
 	const panzoomInstance = panzoom(svg, {
-		minZoom: 0.5,
-		maxZoom: 100,
+		transformOrigin: { x: 0.5, y: 0.5 },
 		autocenter: true,
 		zoomDoubleClickSpeed: 1,
 		smoothScroll: false,
-		onTouch: function () {
+		initialX: (svg.parentElement?.offsetWidth ?? 0) / 2,
+		initialY: (svg.parentElement?.offsetHeight ?? 0) / 2,
+		initialZoom: 0.85,
+		zoomSpeed: 0.05,
```

Comment:
I don't know if this is required for iOS or anything, but this is really slow on desktop. Could we up this again?